guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,184 @@
+[id="Upgrading-Project-from-EL7-to-EL8{context}"]
+= Upgrading {Project} from {RHEL} 7 to {RHEL} 8
```

Comment:
For Foreman, this renders as "Upgrading Foreman from Red Hat Enterprise Linux 7 to Red Hat Enterprise Linux 8", which is exactly the combination we do not support today.

Can we make it render "Upgrading Satellite from Red Hat Enterprise Linux 7 to Red Hat Enterprise Linux 8", but for the others it'd just be "Enterprise Linux" without the "Red Hat", as that would include CentOS and friends.